--- debian/rules.old
+++ debian/rules
@@ -11,9 +11,6 @@
 # upsteram maintains config.h.in manually
 export AUTOHEADER = true
 
-override_dh_auto_configure:
-	dh_auto_configure -- --disable-readline
-
 override_dh_auto_test:
 ifneq (,$(filter $(DEB_HOST_ARCH_OS), linux))
 	dh_auto_test
@@ -24,4 +21,4 @@
 %:
 	dh $@ --with=autoreconf
 
-.PHONY: override_dh_auto_configure override_dh_auto_test
+.PHONY: override_dh_auto_test
--- debian/control.old
+++ debian/control
@@ -2,7 +2,7 @@
 Section: net
 Priority: optional
 Maintainer: Laszlo Boszormenyi (GCS) <gcs@debian.org>
-Build-Depends: debhelper-compat (= 13), libssl-dev, libwrap0-dev, openssl <!nocheck>, net-tools <!nocheck>, iproute2 [linux-any] <!nocheck>, netbase <!nocheck>, procps <!nocheck>
+Build-Depends: debhelper-compat (= 13), libssl-dev, libwrap0-dev, libreadline-dev, openssl <!nocheck>, net-tools <!nocheck>, iproute2 [linux-any] <!nocheck>, netbase <!nocheck>, procps <!nocheck>
 Standards-Version: 4.6.2
 Rules-Requires-Root: no
 Homepage: https://www.dest-unreach.org/socat/

